euth/organisations/templates/euth_organisations/organisation_detail.html

```diff
@@ -6,9 +6,17 @@
         <div class="container-narrow">
             <div class="euth-hero-unit-header">
                 <ul class="orgs-social-links">
-                    <li><a href="#"><i class="icon-facebook"></i></a></li>
-                    <li><a href=""><i class="icon-twitter"></i></a></li>
-                    <li><a href=""><i class="icon-instagram"></i></a></li>
+                  {% if organisation.handle_facebook  %}
+                  <li><a href="https://www.facebook.com/{{organisation.handle_facebook}}"><i class="fa fa-facebook"></i></a></li>
+                  {% endif %}
+                  {% if organisation.handle_twitter %}
+                  <li><a href="https://twitter.com/{{organisation.handle_twitter}}"><i class="fa fa-twitter"></i></a></li>
+                  {% endif %}
+                  {% if organisation.handle_instagram %}
+                  <li><a href="https://www.instagram.com/{{organisation.handle_instagram}}"><i class="fa fa-instagram"></i></a></li>
+                  {% endif %}
+                  {% if organisation.webpage %}<li><a href="{{organisation.webpage}}"><i class="fa fa-link"></i></a></li>
+                  {% endif %}
                 </ul>
```

Comment:
Is there a way to make sure, the <ul> won't be empty if there's none of the fields used? Probably only way to check is to repeat all the {% if %} together inside one if around the whole <ul>, mh?